sphinx/tutorial/rose/furthertopics/rose-bunch.rst

```diff
@@ -180,17 +180,26 @@ to indexed files, it can sometimes be difficult to quickly identify which file
 to look in for output. To aid this, :rose:app:`rose_bunch` supports naming
 command instances via the :rose:conf:`rose_bunch[bunch]names=` option.
 
+Returning to our source directory::
+
+   cd ~/cylc-src/rose-bunch
+
 Open your app config (under ``app/lander/rose-app.conf``) and add the
 following line under the :rose:conf:`rose_bunch[bunch]` section:
 
 .. code-block:: rose
 
    names=BA123 Emirates345 BA007 PC456
 
-Re-run your suite and, once it has finished, open up Cylc Review and examine
-the job listing. In the drop-down ``bunch.*.err`` and ``bunch.*.out`` boxes you
-should now see entries for the names you've configured rather than the
-``bunch.0.out ... bunch.3.out`` entries previously present.
+We can now install and play this updated suite::
+
+   cylc install rose-bunch
+   cylc play rose-bunch
+   cylc gui
```

Comment:
TL;DR - this is definitely still the territory of Cylc (7) Review. Like the new Cylc GUI, we don't want to tell people to start a new instance of Cylc Review each time. If they are a power user they will know how to use `cylc review start` and if not someone will hopefully have set up a server for them like we do locally.

Also: I've been working on the basis that we don't tell people to open the GUI like this, because
- We don't want them to do it every time they play workflow
- They might prefer to use TUI, or have no choice (TUI is bundled, GUI is an add-on)

At the moment, however, you cannot easily link to the appropriate page in the Cylc tutorial.

Reply:
Great, thanks @wxtim, maybe https://github.com/metomi/rose/issues/2541 can swallow this then. I'll remove references for now.